x-pack/plugin/esql/compute/src/main/java/org/elasticsearch/compute/operator/X-MultivalueDedupe.java.st

```diff
@@ -50,9 +46,7 @@ $if(BytesRef)$
     private static final int ALWAYS_COPY_MISSING = 20;  // TODO BytesRef should try adding to the hash *first* and then comparing.
 $elseif(double)$
     private static final int ALWAYS_COPY_MISSING = 110;
-$elseif(int)$
-    private static final int ALWAYS_COPY_MISSING = 300;
-$elseif(long)$
+$else$
     private static final int ALWAYS_COPY_MISSING = 300;
```

Comment:
I'd intended to make this something you'd have to fill out. Maybe I should have had an `else` section that says "you have to benchmark this. With actual instructions on how. Like, there's a benchmark in `MultivalueDedupeBenchmark` but I don't 100% remember how I did the benchmarking.

Reply:
It is likely long and int would behave all that differently?